dashboard/app/views/projects/index.html.haml

```diff
@@ -69,6 +69,10 @@
       $scope.projectsLoaded = true;
     }).catch($scope.genericError);
 
+    $scope.projectVisible = function(project) {
+      return (!project.hidden);
```

Comment:
I could see an API where fetches for the hidden projects would still succeed, they just wouldn't be enumerated when querying.